query-engine/query-structure/src/default_value.rs

```diff
@@ -182,23 +182,21 @@ impl ValueGenerator {
         ValueGenerator::new("now".to_owned(), vec![]).unwrap()
     }
 
-    pub fn new_cuid() -> Self {
-        ValueGenerator::new("cuid".to_owned(), vec![]).unwrap()
+    pub fn new_cuid(version: u8) -> Self {
+        ValueGenerator::new("cuid".to_owned(), vec![(None, PrismaValue::Int(version as i64))]).unwrap()
```

Comment:
The first argument of the tuple in `Vec<>` seems to be totally useless, btw. This is something we can clean up after Prisma 6